lookout/style/typos/analyzer.py

```diff
@@ -1,20 +1,34 @@
 """Identifier typos analyzer."""
 from collections import defaultdict
 import logging
-from typing import Any, Dict, List, Mapping, Tuple
+from typing import Any, Dict, Iterable, Iterator, List, Mapping, NamedTuple, Sequence, Tuple
 
 import bblfsh
 from lookout.core.analyzer import Analyzer, AnalyzerModel, DummyAnalyzerModel, ReferencePointer
 from lookout.core.api.service_analyzer_pb2 import Comment
 from lookout.core.data_requests import DataService, \
     with_changed_uasts_and_contents, with_uasts_and_contents
 from lookout.core.lib import extract_changed_nodes, files_by_language, filter_files, find_new_lines
+from lookout.sdk.service_data_pb2 import Change, File
 import pandas
 from sourced.ml.algorithms import TokenParser, uast2sequence
 
+from lookout.style.format.utils import generate_comment
 from lookout.style.typos.corrector_manager import TyposCorrectorManager
 from lookout.style.typos.utils import Columns, flatten_df_by_column
 
+Candidate = NamedTuple("Candidate", (
```

Comment:
I think this class should be moved to the `utils.py` in the future and used everywhere through corrector suggestions logic. I can do it after this PR is merged

Reply:
Ok, I'll move it to `utils.py`.